### PR TITLE
graphpath 1.0 (new formula)

### PR DESCRIPTION
**Pull request by cjwilburn**
- [ x ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ x ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ x ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
This PR adds a formula for `graphpath`.

### CODE REVIEW

**Comment by cjwilburn** [comment]
It looks like there are some commits beyond what's bundled in the `graphpath` release as `v1.0`. I'll try to put in a PR on the original executable to rectify this.

File: Formula/graphpath.rb [hunk 1]
```diff
@@ -1,8 +1,14 @@
 class Graphpath < Formula
   desc "Generates ASCII network diagram from the route table of a Unix/Linux"
   homepage "https://bsdrp.net/"
+<<<<<<< HEAD
```
Comment (SMillerDev) [inline Formula/graphpath.rb]: This doesn't seem right

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/graphpath.rb (hunk 1 above)
Comment (cjwilburn) [reply Formula/graphpath.rb]: Do you mean the `homepage` URI or `<<<<<<< HEAD`? The latter has been removed in subsequent commits.

Comment (SMillerDev) [reply Formula/graphpath.rb]: I meant the merge conflict

Comment (cjwilburn) [reply Formula/graphpath.rb]: I was pushing up an update to the tarball destination when I encountered the merge conflict and resolved it. What about the merge conflict looks off?

**Comment by SMillerDev** [comment]
Is there any way you can check that the software actually works? (Instead of just printing the version)

**Comment by cjwilburn** [comment]
@SMillerDev I've fixed the smoke test to give an example graph, and you should see  IPv6 support with this version, as well.

**Comment by cjwilburn** [comment]
@SMillerDev would it be better for me to mitigate any problems with this on this PR or open a new one?

**Comment by igas** [comment]
@BrewTestBot test this please

**Comment by cjwilburn** [comment]
I'm going to rebase my work over the latest upstream changes to see if that fixes the conflicts.

**Comment by igas** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Please squash everything into a single commit, rebased on master.

**Comment by cjwilburn** [comment]
> Please squash everything into a single commit, rebased on master.

I did, but in two steps, so things may look a bit goofy. I can redo this if Jenkins is still having problems reconciling the commits.

**Comment by fxcoudert** [comment]
- Remove `head`
- You have an unrelated `carla` formula change

**Pull request closed without merging**